letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    "A"
+  elsif integer < 90 && integer >= 80
+    "B"
+  elsif integer < 80 && integer >= 70
+    "C"
+  elsif integer < 70 && integer >= 60
+    "D"
+  else
+    "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort_by {|word| word.length}.first
```

Comment:
'.min_by' is a handy little ruby function that will make this a bit more concise.

https://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-min_by